README.md

```diff
@@ -55,24 +55,25 @@ A test case might look something like this:
         description='Create Load Balancer: ' + load_balancer_name,
         application=bindings['TEST_APP'])
 
+    hc_match_spec = {key: jp.NUM_EQ(value) for key, value in spec.items()}
+    hc_match_spec['name'] = jp.STR_SUBSTR('%s-hc' % load_balancer_name)
     builder = gcp_testing.GceContractBuilder(self.gce_observer)
     (builder.new_clause_builder('Health Check Added',
                                 retryable_for_secs=30)
      .list_resources('http-health-checks')
-     .contains_pred_list(
-         [json_predicate.PathContainsPredicate(
-              'name', '%s-hc' % load_balancer_name),
-          json_predicate.DICT_SUBSET(spec)]))
+     .contains_match(hc_match_spec)
+
     (builder.new_clause_builder('Target Pool Added',
                                 retryable_for_secs=30)
      .list_resources('target-pools')
      .contains_path_value('name', '%s-tp' % load_balancer_name))
     (builder.new_clause_builder('Forwarding Rules Added',
                                 retryable_for_secs=30)
      .list_resources('forwarding-rules')
-     .contains_pred_list([
-          json_predicate.PathContainsPredicate('name', load_balancer_name),
-          json_predicate.PathContainsPredicate('target', target_pool_name)]))
+     .contains_match({
```

Comment:
I'm not sure `contains_match` makes a lot of sense for what this function does, which is validating predicates based on attribute paths.